.travis.sh

```diff
@@ -22,6 +22,7 @@ make tests-pmtud
 make tests-tap
 make tests-ip-fragment
 make tests-thread
+make doc
```

Comment:
Does travis will fail in case of Warning or Error during doxygen generation ? Did you test it ?